hathor/feature_activation/resources/feature.py

```diff
@@ -0,0 +1,144 @@
+#  Copyright 2023 Hathor Labs
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from typing import Optional
+
+from hathor.api_util import Resource, set_cors
+from hathor.cli.openapi_files.register import register_resource
+from hathor.feature_activation.feature import Feature
+from hathor.feature_activation.feature_service import FeatureService
+from hathor.feature_activation.model.feature_state import FeatureState
+from hathor.feature_activation.settings import Settings
+from hathor.transaction.storage import TransactionStorage
+from hathor.utils.api import Response
+
+
+@register_resource
+class FeatureResource(Resource):
+    __slots__ = ()
+
+    isLeaf = True
+
+    def __init__(
+        self,
+        *,
+        settings: Settings,
```

Comment:
I think using simply `settings` with type `Settings` might be very misleading and error prone. Maybe rename it to `feature_settings`?

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/634/commits/6a8d8ab3d82cbd99c98536abb421644cab724364

Reply:
Great! Thanks!